pallets/delegation/src/lib.rs

```diff
@@ -485,7 +485,8 @@ impl<T: Config> Pallet<T> {
 		Ok((revocations, consumed_weight))
 	}
 
-	// Revoke all children of a delegation
+	/// Revokes all children of a delegation.
+	/// Returns the number of revoked delegations and the consumed weight.
 	fn revoke_children(
```

Comment:
At this point, we don't use the result of revoke_children anywhere. Should we remove it?

Reply:
Or is it needed for weight calculation?

Reply:
we do!

Reply:
```rust
			Self::revoke_children(delegation, sender, remaining_revocations).map(|(r, w)| {
				revocations = revocations.saturating_add(r);
				consumed_weight = consumed_weight.saturating_add(w);
			})?;
```

Reply:
True! ✌🏻